testsuite/integration-arquillian/tests/base/src/test/resources/META-INF/keycloak-server.json

```diff
@@ -280,7 +280,7 @@
     },
 
     "userProfile": {
-        "provider": "${keycloak.userProfile.provider:declarative-user-profile}",
+        "provider": "${keycloak.userProfile.provider:}",
```

Comment:
When we don't specify the default provider here, we can test the default settings for distribution of Keycloak Server and if the default provider is not set up, the tests will fail, which is expected and will be revealed earlier.